inductiva/simulators/seissol.py

```diff
@@ -0,0 +1,58 @@
+"""Class to run SeisSol simulations on a custom container image."""
+
+from typing import List, Optional
+from inductiva import types, tasks, simulators
+
+
+@simulators.simulator.mpi_enabled
+class SeisSol(simulators.Simulator):
+    """Class to run SeisSol simulations on a custom container image."""
```

Comment:
If we are implementing a class for SeisSol on the python client we intend that class to use our docker images and not a custom docker image.